docs/user-guide/advanced/replicate_evaluations.md

```diff
@@ -140,6 +140,30 @@ IN_FEATURES=1024 \
 eva predict_fit --config configs/vision/pathology/offline/<task>.yaml
 ```
 
+### hibou-B (hist.ai) - DINOv2 ViT-B14 (1M Slides) [[7]](#references)
+To evaluate [hist.ai's](https://www.hist.ai/) FM with DINOv2 ViT-B14 backbone, pretrained on
+a proprietary dataset of one million slides, run: 
```

Comment:
add link to model card, e.g.: `.... slides, available for download on [huggingface](https://huggingface.co/histai/hibou-b), run:`

Reply:
done